go/sign/access_key_singer_test.go

```diff
@@ -0,0 +1,40 @@
+package sign
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strings"
+	"time"
+)
+
+func ExampleAccessKeyHTTPSigner_Sign() {
+	req, _ := http.NewRequest("GET",
+		"https://gateway-staging.ncrcloud.com/site/sites/find-nearby/88.05,46.25?radius=10000",
```

Comment:
I would externalize this URL to not have to require a code change if the value is ever changed in the future